packages/fiori/src/ShellBarTemplate.tsx

```diff
@@ -0,0 +1,275 @@
+									aria-label={this._searchBtnOpen}
+									aria-expanded={this._searchFieldExpanded}
+									accessibilityAttributes={this.accInfo.search.accessibilityAttributes}
+								/>
+							</>
+						)}
+						{this.hasAssistant && (
+							<div id="assistant" class={this.classes.assistant}>
+								<slot name="assistant"></slot>
+							</div>
+						)}
+						{this.showNotifications && (
+							<Button
+								id={`${this._id}-item-2`}
+								class={{
+									"ui5-shellbar-button": true,
+									"ui5-shellbar-bell-button": true,
+									"ui5-shellbar-items-for-arrow-nav": true,
+									...this.classes.notification,
+								}}
+								icon="sap-icon://bell"
+								data-ui5-text="Notifications"
+								data-ui5-notifications-count={this.notificationsCount}
+								onClick={this._handleNotificationsPress}
+								tooltip={this._notificationsText}
+								accessibilityAttributes={this.accInfo.notifications.accessibilityAttributes}
+								data-ui5-stable="notifications"
+							/>
+						)}
+						{this.customItemsInfo.map(item => (
+							<Button
+								key={item.id}
+								id={item.id}
+								class={`${item.classes} ui5-shellbar-items-for-arrow-nav`}
+								icon={item.icon}
+								tooltip={item.tooltip}
+								data-count={item.count}
+								data-ui5-notifications-count={this.notificationsCount}
+								data-ui5-external-action-item-id={item.refItemid}
+								data-ui5-stable={item.stableDomRef}
+								onClick={item.press}
+							/>
+						))}
+						<Button
+							id={`${this._id}-item-5`}
+							class={{
+								"ui5-shellbar-no-overflow-button": true, // always visible
+								"ui5-shellbar-button": true,
+								"ui5-shellbar-overflow-button": true,
+								...this.classes.overflow,
+							}}
+							icon="sap-icon://overflow"
+							data-count={this._overflowNotifications}
+							onClick={this._handleOverflowPress}
+							tooltip={this._overflowText}
+							accessibilityAttributes={this.accInfo.overflow.accessibilityAttributes}
+							data-ui5-stable="overflow"
+						/>
+						{this.hasProfile && profileButton.call(this)}
+						{this.showProductSwitch && (
+							<Button
+								id={`${this._id}-item-4`}
+								class={`ui5-shellbar-no-overflow-button ui5-shellbar-button ui5-shellbar-button-product-switch ui5-shellbar-items-for-arrow-nav`}
+								icon="sap-icon://grid"
+								data-ui5-text="Product Switch"
+								onClick={this._handleProductSwitchPress}
+								tooltip={this._productsText}
+								aria-label={this._productSwitchBtnText}
+								aria-haspopup="dialog"
+								aria-expanded={this.accInfo.products.accessibilityAttributes.expanded}
+								accessibilityAttributes={this.accInfo.products.accessibilityAttributes}
+								data-ui5-stable="product-switch"
+							/>
+						)}
+					</div>
+				</div>
+			</header>
+			{ShellBarPopoverTemplate.call(this)}
+		</>
+	);
+}
+
+function profileButton(this: ShellBar) {
+	return (
+		<Button
+			profile-btn
```

Comment:
probably data-ui5-profile-btn, as data- and aria- should be the only cases for setting attributes